### ISSUE TITLE
Suggestion: Make result "copyable"

### ISSUE THREAD

**Issue by Wolf-St** [tomer8007/windows-shazam#3]
Hi Tomer,
First Thanks for the tools ---- great help for me on Win7!
I want to suggest that you may rework the result window in a way that allows for marking and copying the Title and Artist etc. to the clipboard for further use (I.e. paste into Mp3-Tagging software)

Another "nice to have" would be some indication that shazam actually receives any data. Took me hours to realize that my microphone had died! (I have a weird system, where I cannot monitor the mike's output live, and just didn't think of it!)

**Comment by tomer8007** [comment]
These are very nice improvements indeed.
I think that for the microphone part I could put a bar of audio that corresponds to the volume so you could tell it's hearing something.
Copy and paste is even easier to implement.

Maybe I could try doing this soon. Thanks for trying this anyway

**Comment by Patraskon** [comment]
Instead of a microphone, you can use a stereo mixer.

**Comment by tomer8007** [comment]
Yeah, I already have programs to use [WasapiLoopbackCapture](https://github.com/naudio/NAudio/blob/master/Docs/WasapiLoopbackCapture.md) or something similar.

**Comment by tomer8007** [comment]
@Wolf-St By the way, when the message box has the focus you can press Ctrl + C and its contents will be copied.
